### PR TITLE
[fix](auth)Fix create user no permissions of information_schema database

### PR DESCRIPTION
**Pull request by DongLiang-0**
## Proposed changes
When creating a new user, this user does not have the information_schema database permission.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by DongLiang-0** [comment]
run buildall

**Comment by DongLiang-0** [comment]
run buildall

**Comment by DongLiang-0** [comment]
run buildall

File: regression-test/suites/account_p0/test_alter_user.groovy [hunk 1]
```diff
@@ -192,5 +192,15 @@ suite("test_alter_user", "account") {
     result2 = connect(user = 'test_auth_user4', password = '12345', url = context.config.jdbcUrl) {
         sql 'select 1'
     }
+
+    // 9. test user default database privileges
+    sql """drop user test_auth_user4"""
```
Comment (morningman) [inline regression-test/suites/account_p0/test_alter_user.groovy:197]:
```suggestion
    sql """drop user if exists test_auth_user4"""
```

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by DongLiang-0** [comment]
run buildall

**Comment by DongLiang-0** [comment]
run buildall

**Review by zddr — APPROVED** [review]
LGTM

**Comment by DongLiang-0** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by zddr — APPROVED** [review]

**Pull request merged**